File: wifi4eu-portal/wifi4eu-portal-web/src/main/angular/src/app/shared/swagger/model/VoucherAssignmentAuxiliarDTO.ts
import * as models from './models';
import {Type} from "class-transformer";

export interface VoucherAssignmentAuxiliarDTO {
    id?: number;

    executionDate?: number;

    status?: number;

    hasPreListSaved?: boolean;

    preListExecutionDate?: number;

    hasFreezeListSaved?: boolean;

    freezeLisExecutionDate?: number;

    notifiedDate?: number;

}




/*default implementation one might extend from (or use as is) */


export class VoucherAssignmentAuxiliarDTOBase  implements VoucherAssignmentAuxiliarDTO{

    id?:  number ;


    executionDate?:  number ;


    status?:  number ;


    hasPreListSaved?:  boolean ;


    preListExecutionDate?:  number ;


    hasFreezeListSaved?:  boolean ;


    freezeLisExecutionDate?:  number ;


    notifiedDate?:  number ;

}

File: wifi4eu-common/src/main/java/wifi4eu/wifi4eu/common/dto/model/VoucherAssignmentAuxiliarDTO.java
package wifi4eu.wifi4eu.common.dto.model;

public class VoucherAssignmentAuxiliarDTO {

    private Integer id;
    private Long executionDate;
    private Integer status;
    private Boolean hasPreListSaved;
    private Long preListExecutionDate;
    private Boolean hasFreezeListSaved;
    private Long freezeLisExecutionDate;
    private Long notifiedDate;

    public VoucherAssignmentAuxiliarDTO() {

    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Long getExecutionDate() {
        return executionDate;
    }

    public void setExecutionDate(Long executionDate) {
        this.executionDate = executionDate;
    }

    public Integer getStatus() {
        return status;
    }

    public void setStatus(Integer status) {
        this.status = status;
    }

    public Boolean getHasPreListSaved() {
        return hasPreListSaved;
    }

    public void setHasPreListSaved(Boolean hasPreListSaved) {
        this.hasPreListSaved = hasPreListSaved;
    }

    public Long getPreListExecutionDate() {
        return preListExecutionDate;
    }

    public void setPreListExecutionDate(Long preListExecutionDate) {
        this.preListExecutionDate = preListExecutionDate;
    }

    public Boolean getHasFreezeListSaved() {
        return hasFreezeListSaved;
    }

    public void setHasFreezeListSaved(Boolean hasFreezeListSaved) {
        this.hasFreezeListSaved = hasFreezeListSaved;
    }

    public Long getFreezeLisExecutionDate() {
        return freezeLisExecutionDate;
    }

    public void setFreezeLisExecutionDate(Long freezeLisExecutionDate) {
        this.freezeLisExecutionDate = freezeLisExecutionDate;
    }

    public Long getNotifiedDate() {
        return notifiedDate;
    }

    public void setNotifiedDate(Long notifiedDate) {
        this.notifiedDate = notifiedDate;
    }
}
